scss/wordpress.scss

```diff
@@ -0,0 +1,52 @@
+/*****************************************************************************\
+  variables
+\*****************************************************************************/
+
+@import 'variables/disabled';
+
+
+
+
+
+/*****************************************************************************\
+  colors
+\*****************************************************************************/
+
+@import 'colors/black';
```

Comment:
Could we import these colors through a common `_colors.scss` file to avoid duplication and disparity between the main stylesheet files? We are now importing roughly the same set of colors for three separate stylesheets, and it would be easier to maintain under a single file.

Reply:
Absolutely. I thought about doing that, then forgot to follow through. I'll update it.